### DISCUSSION TITLE
Create an Android for Cars compatible App

### DISCUSSION THREAD

**Discussion by cnico** [home-assistant/frontend#8480] [Other feature requests] [answered]
Hi,

It would be really great if the Android app could be used in the Android Auto context. 
I means that if you have a vehicule with a screen that supports Android Auto, you could run Home Assistant from the screen of the car interacting through the smartphone and using all the feature of home assistant's frontend.

I do not know if it is complex technically. I just found the official documentation : https://developer.android.com/cars

I give you the idea and think about it if you include it in a roadmap or not.

Regards,

**Comment by spacegaier** [comment, marked as answer]
That sounds like a discussion / request for https://github.com/home-assistant/android.

**Reply by cnico** [reply]
Your are right, I transfert to android project.
